test/cache_test.rb

```diff
@@ -202,10 +202,12 @@ def test_cache_expiration_in_collection_on_update_of_record
       end
 
       bar = 'Bar'
-      author.update!(name: bar)
+      Timecop.travel(10.seconds.from_now) do
```

Comment:
@iggant can you please elaborate the case why you need **Timecop** in this case?

Reply:
before this change there were random test failture 
probably this is because cache version relay on updated_at column which doesn't contain millisecond part, so time to time it doesn't invalidate cache. 
and with travel to 10 second ahead we eliminate this possibility 
probably current check in the 172 line is also related, but I will thy to change that test in the next PR

Reply:
perfect reason :ok_hand: can't wait to see you apply the same logic for the other test too on line 167. You can do it in your next PR. Then we may also be able to remove these weird checks
```ruby
if ENV['APPVEYOR'] && actual != expected
  skip('Cache expiration tests sometimes fail on Appveyor. FIXME :)')
```